sub/sd_ass.c

```diff
@@ -23,6 +23,7 @@
 
 #include <libavutil/common.h>
 #include <ass/ass.h>
+#include <ass/ass_types.h>
```

Comment:
To be able to test on ASS_JUSTIFY_AUTO.
But I see that libass now has released a new version so I could change to test on LIBASS_VERSION instead and then this include is not needed.